src/Apps/Artwork/Components/OtherWorks/Header.tsx

```diff
@@ -16,14 +17,11 @@ export const Header: React.SFC<HeaderProps> = props => {
         {title}
       </Serif>
       {buttonHref && (
-        <Button
-          variant="secondaryOutline"
-          mb={3}
-          // FIXME: Move to <Link> component once https://github.com/artsy/palette/pull/130 is merged
-          onClick={() => (window.location.href = buttonHref)}
-        >
-          View all
-        </Button>
+        <RouterLink to={buttonHref}>
```

Comment:
Ah cool, yea these custom `window.location.href` navigations in JS dont work with our router.